### DISCUSSION TITLE
Deprecation Warnings

### DISCUSSION THREAD

**Discussion by oam-victor** [upb-lea/gym-electric-motor#213] [Q&A]
I have downloaded the files so that I can try to learn how to make the control of some motors and I tried to run some examples. However, I've been getting some deprecation warnings. I tried to run the classic controller examples but I could only run the DC one. Ind and Sync motors are getting deprecation warnings. I tried to change somethings, but It hasn't worked. At this point, I am afraid of changing the code and getting more errors. Has anyone faced this and understands how to solve it?

**Comment by wkirgsn** [comment]
Hi, thanks for hinting on these warnings.
Can you give as a console output of these warnings for us to better target this issue? Thank you in advance
